### PR TITLE
Remove npm-run-all dependency from where it's not needed

### PR DESCRIPTION
**Pull request by lipis**
There is no need for unnecessary dev dependencies.

### CODE REVIEW

File: integration-tests/gatsby-image/package.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
+    "gatsby-image": "latest",
+    "gatsby-plugin-manifest": "latest",
+    "gatsby-plugin-offline": "latest",
+    "gatsby-plugin-react-helmet": "latest",
+    "gatsby-plugin-sharp": "latest",
+    "gatsby-source-filesystem": "latest",
+    "gatsby-transformer-sharp": "latest",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
     "react-helmet": "^5.2.0"
```
Comment (lipis) [inline integration-tests/gatsby-image/package.json]: also updated to the latest.

**Comment by pieh** [comment]
Those aren't starters - they are not used by users, those are used for e2e tests and we use `npm-run-all` in other places anyway, so not sure if there is value in removing it

**Comment by lipis** [comment]
Overall it's better to have less and we don't gain anything from those here.. yes maybe in other places it makes sense to have the `npm-run-all` but let's keep the simple things.. simple.

**Comment by lipis** [comment]
🍻

**Pull request merged**